ParabolInc/parabol
# feat(kudos): enable mentions in retro reflections

Related to #9234. We need mentions in reflections to later use it for sending kudos

<img width="344" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/978252f5-4fd5-4256-b613-60b9095fd4ed">

**How to test:**

- Start a retro
- Add few reflections and see that you can mention team users in them
- See that in all other places mention is rendered correctly — vote phase, summary etc.
- Because of some refactoring, play with retro and make sure nothing's broken in general for reflection cards or and reflections editor
